### PR TITLE
Updated Plugins in go.mod 

### PR DESCRIPTION
**Pull request by patrickog11**

### CODE REVIEW

**Comment by garyloug** [comment]
Testing these..

File: go.mod [hunk 1]
```diff
@@ -5,8 +5,8 @@ go 1.14
 require (
 	git.fd.io/govpp.git v0.3.5
 	github.com/containernetworking/cni v0.8.0
-	github.com/containernetworking/plugins v0.8.0
-	github.com/coreos/go-iptables v0.4.5 // indirect
+	github.com/containernetworking/plugins v0.8.7
+        github.com/coreos/go-iptables v0.4.5 // indirect
```
Comment (garyloug) [inline go.mod:9]: @patrickog11 can you change whitespace back to a tab here

**Review by garyloug — COMMENTED** [review]
whitespace

**Comment by johnoloughlin** [comment]
outdated

**Pull request closed without merging**